deepmd/train/trainer.py

```diff
@@ -180,6 +180,9 @@ def fitting_net_init(fitting_type_, descrpt_type_, params):
             self.fitting_type = fitting_type
             fitting_param.pop("type", None)
             fitting_param["descrpt"] = self.descrpt
+            fitting_param["compress"] = self.is_compress
+            if self.is_compress and fitting_type != "ener":
+                raise RuntimeError("compression can only be performed in energy model, but " + fitting_type + " is given")
```

Comment:
I don't think there is any reason to limit the energy model, which breaks the existing compression for the tensor models.